apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -73,17 +68,18 @@ const IMPORT_ROSTER_REQUEST = 'teacherSections/IMPORT_ROSTER_REQUEST';
 /** Reports request to import a roster has succeeded */
 const IMPORT_ROSTER_SUCCESS = 'teacherSections/IMPORT_ROSTER_SUCCESS';
 
-/** @const A few action keys exposed for unit test setup */
+/** @const A few constants exposed for unit test setup */
 export const __testInterface__ = {
+  EDIT_SECTION_SUCCESS,
   IMPORT_ROSTER_FLOW_BEGIN,
   IMPORT_ROSTER_FLOW_LIST_LOADED,
+  PENDING_NEW_SECTION_ID,
+  USER_EDITABLE_SECTION_PROPS,
 };
 
 //
 // Action Creators
 //
-export const setStudioUrl = studioUrl => ({ type: SET_STUDIO_URL, studioUrl });
```

Comment:
`studioUrl` is never used, now that all section management that uses redux is running on dashboard.